server/api/router.go

```diff
@@ -134,15 +134,14 @@ func createRouter(prefix string, svr *server.Server) *mux.Router {
 
 	adminHandler := newAdminHandler(svr, rd)
 	router.HandleFunc("/api/v1/admin/cache/region/{id}", adminHandler.HandleDropCacheRegion).Methods("DELETE")
+	router.HandleFunc("/api/v1/admin/reset-ts", adminHandler.ResetTS).Methods("POST")
```

Comment:
How about `POST /api/v1/admin/ts`? An action(`reset` here) is not recommended in the URL.

Reply:
I think reset more meaningful, it's a dangerous action.